apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/PatientLinkBuilder.java

```diff
@@ -66,7 +66,7 @@ public PatientLinkBuilder(PatientLinkBuilder prev, boolean hasAnotherPage) {
    */
   private void validate() {
     if (getPageSize() <= 0) {
-      throw new InvalidRequestException("A zero or negative page size is unsupported");
+      throw new InvalidRequestException("Value for pageSize cannot be negative: %s");
```

Comment:
Zeroes also appear to be invalid here.

Reply:
Good catch; not sure why the patient paging doesnt allow 0 and every other page size does, but I'll just adjust the message here to catch that and not second guess the implementation.